classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
```

Comment:
- [ ] Please replace `require` with `require_relative`. `require` is ideally used when you want to include dependencies, while `require_relative` is ideally suited to when you need to bring in your code from another file in the same project 😎